packages/leafygreen-provider/README.md

```diff
@@ -31,7 +31,7 @@ This is the context provider for LeafyGreen Provider, and should be added as hig
 ```JS
 import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
 
-<LeafyGreenProvider>
+<LeafyGreenProvider baseFontSize={14}>
```

Comment:
I think we want to encourage people to use the default if possible. Since we default to 14, this is superfluous and probably not a recommended implementation.

Reply:
👍